src/components/TutorialPage/index.tsx

```diff
@@ -283,17 +283,20 @@ const TutorialPage = (): JSX.Element => {
                     <h1 id="browser-support">Browser support</h1>
                     <ul>
                         <li>
-                            Currently, Simularium supports Firefox, Chrome, and
-                            Edge. Some features may not work on other browsers.
+                            Currently, Simularium supports Firefox, Safari,
+                            Chrome, and Edge. Some features may not work on
+                            other browsers.
                         </li>
                         <li>
-                            If using Safari on a Mac, please enable WebGL 2.0 by
-                            choosing Develop &gt; Experimental Features and
-                            enabling &quot;WebGL 2.0&quot; (If you do not have a
-                            Develop menu in your menu bar, please first choose
+                            If using Safari on a Mac and it is not working,
```

Comment:
Nit: This reads weirdly to me. I'd add something like "If you are using Safari on a Mac and ~~it is not working~~ the content below is not loading", or whatever the expected issue would be.

Reply:
@toloudis do you know what the behavior would be if webGl isn't enabled? do they get an error message or is the viewer blank?

Reply:
We have a webgl1 fallback from webgl2, but if they don't have WebGL at all that's a huge problem. I have no idea what happens then.   I think even webgl2 is supported on well over 90% of devices by now.